lib/jekyll-seo-tag/json_ld_drop.rb

```diff
@@ -84,7 +84,11 @@ def main_entity
       private :main_entity
 
       def to_json
-        to_h.reject { |_k, v| v.nil? }.to_json
+        drop_hash = to_h
+        if page_drop.custom_structured_data.is_a? Hash
+          drop_hash = Utils.deep_merge_hashes(drop_hash, page_drop.custom_structured_data)
+        end
+        drop_hash.reject { |_k, v| v.nil? }.to_json
```

Comment:
@DarkWiiPlayer This plugin now requires at least Ruby 2.4.0.
So, could you bring the branch up to date and use `drop_hash.compact` here?